### DISCUSSION TITLE
Nocode suggestion ( free tier or open source) to build successive popups ?

### DISCUSSION THREAD

**Discussion by billmetangmo** [nextapps-de/winbox#148] [Q&A]
Hello,

_Not sure if this is the best place to ask but I take my chance._
I created 3 successive popups with a next button using pendo.io nocode builder ( because I am null at UI):

![3_successive_popups](https://user-images.githubusercontent.com/25366207/198158815-3f4f1c10-e298-46c1-aebf-fa1f7fa65132.gif)

But the tool will be to expensive for my use case because I just use 0.5% of the product.
I wanted to know if someone knows a nocode specific tool ( free tier or open source)  that i can use to do it .
**Knowing that my constraint is that it should be a just a snippet i have to inject in head of my product because it's the only part I can modify from the tool** 

Any help will be appreciated :pray:

**Comment by ts-thomas** [comment]
I understand your goal, but this can't be done by Winbox without any code. Yes, it could be solved by inserting code into the head section, but then you didn't have something which fits into your no-code stack. Using custom code in no-code environments is just useful when fully understanding the code or when the code just apply passively like a patch. But in this example it isn't, because it looks like a workflow which required full integration into your no-code environment.
